app/controllers/application_controller.rb

```diff
@@ -40,4 +40,10 @@ def redirect_handler(redirect_path, message)
     flash[:notice] = message
   end
 
+  def null_value_check(value , path, message)
+    unless value.present?
+      redirect_handler(path, message)
+    end
+  end
```

Comment:
Instead of this you can raise an exception and rescue that exception here.

Reply:
Done